src/figuresClasses.ts

```diff
@@ -1,11 +1,70 @@
-export interface Figure {}
+export interface Figure {
+  shape: string;
+  color: string;
+  getArea(): number;
+}
+
+function roundDown(num: number): number {
+  return Math.floor(num * 100) / 100;
+}
 
-export class Triangle implements Figure {}
+export class Triangle implements Figure {
+  constructor(
+    public color: string,
+    public a: number,
+    public b: number,
+    public c: number,
+    public shape: string = 'triangle',
```

Comment:
[CODE KNOWLEDGE] - The constructor parameters `color`, `a`, `b`, `c`, and `shape` are being manually assigned to class properties. However, with TypeScript, you don't need to do this manually when you use the `public` keyword in the constructor. TypeScript will automatically create and assign properties with the same names.